bin/setup_psqlgraph.py

```diff
@@ -76,57 +86,84 @@ def create_tables(host, user, password, database):
     """
     create a table
     """
-    print('Creating tables in test database')
+    print("Creating tables in test database")
 
-    engine = create_engine("postgres://{user}:{pwd}@{host}/{db}".format(
-        user=user, host=host, pwd=password, db=database))
+    engine = create_engine(
+        "postgres://{user}:{pwd}@{host}/{db}".format(
+            user=user, host=host, pwd=password, db=database
+        )
+    )
     create_all(engine)
     versioned_nodes.Base.metadata.create_all(engine)
 
 
 def create_indexes(host, user, password, database):
-    print('Creating indexes')
-    engine = create_engine("postgres://{user}:{pwd}@{host}/{db}".format(
-        user=user, host=host, pwd=password, db=database))
+    print("Creating indexes")
+    engine = create_engine(
+        "postgres://{user}:{pwd}@{host}/{db}".format(
+            user=user, host=host, pwd=password, db=database
+        )
+    )
     index = lambda t, c: ["CREATE INDEX ON {} ({})".format(t, x) for x in c]
     for scls in Node.get_subclasses():
         tablename = scls.__tablename__
-        map(engine.execute, index(
-            tablename, [
-                'node_id',
-            ]))
-        map(engine.execute, [
-            "CREATE INDEX ON {} USING gin (_sysan)".format(tablename),
-            "CREATE INDEX ON {} USING gin (_props)".format(tablename),
-            "CREATE INDEX ON {} USING gin (_sysan, _props)".format(tablename),
-        ])
+        list(map(engine.execute, index(tablename, ["node_id",])))
```

Comment:
Aren't all these list()s also unnecessary?

Reply:
in python 2 `map` returned a list, but in python 3 it returns an iterator. the elements are only processed when used, so we need to cast to list to force all elements to be processed

Reply:
Aha of course! Thanks :P